tests/Feature/CreateTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Feature;
+
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Artisan;
+use Tests\TestCase;
+
+class CreateTest extends TestCase
+{
+    use DatabaseMigrations;
+    protected function setUp(): void
+    {
+        parent::setUp();
+        Artisan::call('passport:install');
+    }
+
+    /**
+     * test
+     */
+    public function testRegister(): void
+    {
+        $data = ['email' => 'goof@gmail.com', 'name' => 'san', 'password' => '12345678'];
+
+        $response = $this->json('POST', 'api/register', $data)->assertStatus(201);
```

Comment:
201 - can and must be replaced with constant `ResponseAlias::HTTP_CREATED`

there are also other HTTP codes like:

`ResponseAlias::HTTP_OK -> 200`
`ResponseAlias::HTTP_UNPROCESSABLE_ENTITY -> 422`

and so on